docs/dev/new_check_howto.md

```diff
@@ -270,15 +270,91 @@ Run only the integration tests for faster iterations:
 tox -e integration
 ```
 
-The check is almost done. Let's add the final touches.
+The check is almost done. Let's add the final touches by adding the integration configurations.
 
-## Final touches
+## Integration configuration
+### Integration Check configuration file
+#### Parameters
 
-### Populate the README
+Parameters in a configuration file follow these rules:
+
+* Placeholders should always follow this format: `<THIS_IS_A_PLACEHOLDER>`according to the documentation [contributing guidelines][16]: 
+* All required parameters are **not** commented by default.
+* All optional parameters are commented by default.
+* If a placeholders has a default value for an integration (like the status endpoint of an integration), it can be used instead of just a canva  placeholder.
+
+#### Parameters documentation
+
+Each parameter in your integration check `conf.yaml.example` configuration file, must have a paragraph in form of YAML comment strings with the following format:
+
+```
+## @<COMMAND_1> <ARG_COMMAND_1>
+## @<COMMAND_2> <ARG_COMMAND_2>
+## <DESCRIPTION>
+#
+<YAML_PARAM>: <PLACEHOLDER>
+```
+
+This paragraph contains **commands** which take the form `@command`. A command is valid only when the comment line containing it starts with a double `#` char:
+
+```
+## @command this is valid
+
+# @command this is not valid and will be ignored
+```
+
+`<DESCRIPTION>` is the description of the parameter. It can span across multiple lines in a special comment block.
+
+##### Available commands
+###### Param
+
+The `@param` command aims to describe the parameter for documentation purposes.
+
+```
+@param <name> - <type> - <required> - default:<defval>
+```
+
+Arguments:
+
+* `name`: the name of the parameter, e.g. `apache_status_url`
+* `type`: the data type for the parameter value. Possible values:
+  * integer 
+  * double 
+  * string 
+  * comma separated list of <integer|double|string>
+* `required`: whether the parameter is required or not. Possible values: 
+    * required
+    * optional
+* `defval`: default value for the parameter, can be empty.
+
+The final layout looks like this:
+
+```
+## @param <name> - <type> - <required/optional> - default:<defval>
```

Comment:
Since we already have a "generic" example of a special block like this, I wonder if we should remove this and just show the real world example from apache